package.json

```diff
@@ -148,20 +148,22 @@
     "cors": "^2.8.4",
     "ethereumjs-util": "^5.2.0",
     "express-handlebars": "^3.0.0",
+    "express-locale": "^1.0.5",
     "hh-mm-ss": "^1.2.0",
     "hlsjs-ipfs-loader": "github:ya7ya/hlsjs-ipfs-loader#bd890b9e78f2c273fcb1dc4e1d43ad0afc69558c",
     "immutable": "^3.8.2",
     "ipfs": "github:ya7ya/js-ipfs#paratii/v0.27.6",
     "ipfs-bitswap": "^0.19.0",
     "js-cookie": "^2.2.0",
     "lodash.debounce": "^4.0.8",
+    "node-polyglot": "^2.2.2",
     "nodemailer": "^4.6.4",
     "paratii-js": "github:Paratii-Video/paratii-js#a6e9ea8c24768f86650b18d92d2946d988fd860b",
     "paratii-mediaplayer": "3.1.2",
     "player.js": "^0.1.0",
     "promise-polyfill": "^7.1.2",
     "query-string": "^6.0.0",
-    "react": "16.2.0",
+    "react": "16.3",
```

Comment:
This upgrade is needed to take advantage of the new [context API](https://reactjs.org/docs/context.html), which we take advantage of to inject the translator into components no matter how deeply nested they are